project board icon

Gayle-Thompson-Igwebike opened the pull request:
#22-ProjectBoard-Gayle-Thompson-Igwebike

Added github black logo for project board



## Related to #22 

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #22 

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

Gayle-Thompson-Igwebike committed:
project board icon

Gayle-Thompson-Igwebike committed:
app.js tidy

Gayle-Thompson-Igwebike committed:
classname update

Gayle-Thompson-Igwebike committed:
create css

Gayle-Thompson-Igwebike committed:
app js update

AndrianaOS reviewed (changes requested):
Im not really sure how this is happening based on the CSS provided. Most likely from the units for the height and width. Use a different unit like px or em. vh depends on the viewing screen size.

<img width="1440" alt="Screenshot 2023-08-11 at 12 36 35 pm" src="https://github.com/Gayle-Thompson-Igwebike/GOOD-PR-v1/assets/113118206/5cca707f-4843-401c-bd1d-acf2258327b8">

Gayle-Thompson-Igwebike committed:
New project board image

Add new img for project board as per feedback.

Gayle-Thompson-Igwebike committed:
update Github img and css

Gayle-Thompson-Igwebike committed:
change of icon and css

Gayle-Thompson-Igwebike committed:
Merge branch 'dev' into 22-github-project-board-component-writing-board-icon

Gayle-Thompson-Igwebike commented:
I made updates to this file following Karleen's comment this morning. I added a new image to represent the project board and changed the css style sizing to px as suggested by Andriana.

Gayle-Thompson-Igwebike committed:
Merge branch 'dev' into 22-github-project-board-component-writing-board-icon

Gayle-Thompson-Igwebike committed:
Merge branch 'dev' into 22-github-project-board-component-writing-board-icon
